### ISSUE TITLE
forward() got an unexpected keyword argument 'step'

### ISSUE THREAD

**Issue by Wheat2018** [jieqin-ai/AMR#2]
Hello, there was an exception occurred when I run eval script with command line `python run_sample.py --make_cam_pass True --eval_cam_pass True`.

`TypeError: forward() got an unexpected keyword argument 'step'`

The `forward` method of class `CAM` seemingly did not supports a parameter named `step`.

![image](https://user-images.githubusercontent.com/40771629/148976774-dc4ffe76-f2ce-4b3c-99a8-308617bcb61a.png)

But the `make_came` step had provided it.

![image](https://user-images.githubusercontent.com/40771629/148977207-bf619939-4166-4dce-a912-23a029930aef.png)

I'm wondering what's the effect of `step`, and what should I do for the code.

**Comment by jieqin-ai** [comment]
I have updated the code. Please retry it.

**Comment by jieqin-ai** [comment]
I have updated the code. Please retry it.

**Comment by Wheat2018** [comment]
Yes, it works. Thanks. But now I'm still waiting for the end of the eval. It does have no progress indicator, does it?

**Comment by jimmy15923** [comment]
Hi @Wheat2018 !

I am also running the `make_cam_pass True` but the does not show any progress for over 24 hours. However, the GPU is actually running. Have you successfully generate the CAMs?

**Comment by Wheat2018** [comment]
Nope, the code ran over 48h, and then I had to killed it for some reasons.

**Comment by wxqlab** [comment]
Hi, @jimmy15923 and @Wheat2018 , I'm also having the same problem, and do you know why and have the solution？
